.github/composite-actions/pw-a11y/action.yml

```diff
@@ -0,0 +1,21 @@
+name: 'Accessibility checking (v2)'
+description: 'Build and serve Storybook pages then check accessibility'
+runs:
+  using: composite
+  steps:
+    - name: Install
+      uses: ./.github/composite-actions/install
+
+    - name: Install Playwright Browsers
+      shell: bash
+      run: npx playwright install --with-deps
+
+    - name: Run tests
```

Comment:
```suggestion
    - name: Run a11y tests
```

Reply:
This name would appear at a very nested depth. Not sure it is so important, but nothing blocking for me.